src/include/duckdb/function/scalar/list/contains_or_position.hpp

```diff
@@ -81,6 +87,48 @@ static void TemplatedContainsOrPosition(DataChunk &args, Vector &result, bool is
 	}
 }
 
+template <LogicalTypeId RETURN_TYPE>
+unique_ptr<FunctionData> ListContainsOrPositionBind(ClientContext &context, ScalarFunction &bound_function,
```

Comment:
Does this need to be moved to the header? Can't we just call `ListContainsFun::GetFunction` in `contains` to add it to the function set?